HeterogeneousCore/AlpakaUtilities/interface/alpakastdAlgorithm.h

```diff
@@ -0,0 +1,72 @@
+#ifndef AlpakaCore_alpakastdAlgorithm_h
+#define AlpakaCore_alpakastdAlgorithm_h
+
+#include <algorithm>
+#include <functional>
+#include <utility>
+
+#include <alpaka/alpaka.hpp>
+
+// reimplementation of std algorithms able to compile with Alpaka,
+// mostly by declaring them constexpr
+
+namespace alpaka_std {
+
+  template <typename T = void>
+  struct less {
+    ALPAKA_FN_HOST_ACC constexpr bool operator()(const T &lhs, const T &rhs) const { return lhs < rhs; }
```

Comment:
According to [cppreference](https://en.cppreference.com/w/cpp/utility/functional/less), `std::less::operator()` has been `constexpr` since C++14. Is a separate implementation still needed?